moviepy/video/io/ffmpeg_writer.py

```diff
@@ -81,6 +84,7 @@ def __init__(
         fps,
         codec="libx264",
         audiofile=None,
+        audio_codec=None,
```

Comment:
why not default to "copy" directly here ?

Reply:
Thanks for your question! The default value of audio_codec in VideoClip.write_videofile is None, so I also adapted FFMPEG_VideoWriter to that.
https://github.com/Zulko/moviepy/blob/e7cbfd38615fa265e94b20a47e4d21fc473001ed/moviepy/video/VideoClip.py#L209

This pull request intends that in VideoClip.write_videofile, if a filename is given for the audio argument and a codec name is given for audio_codec, then it will be encoded according to that codec, but if audio If audio_codec=None, I thought it would be better to have it "copied" as before.